_docs/_latest/configure/notifier/index.md

```diff
@@ -200,12 +200,12 @@ notifier:
 Forseti Security can configured to send violations to
 [Cloud Security Command Center (Cloud SCC)](https://cloud.google.com/security-command-center/).
 
-Cloud SCC API is now in public beta. Please see the steps below to setup
-and configure. The previous alpha API will no longer be supported for setup.
+Cloud SCC API is now Generally Available (GA). Alpha and Beta APIs have been 
```

Comment:
Can you please remove `Alpha and Beta APIs have been deprecated.` Just saying `GA` should be enough.